source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,100 @@
+.. _csharp-time-series:
+
+=======================
+Time Series Collections
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use and interact with **time series collections** in
+MongoDB using the {+driver-long+}.
+
+Time series collections efficiently store sequences of measurements over
+a period of time. Time series data consists of any data collected over
+time, metadata that describes the measurement, and the time of the
+measurement.
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Example
+     - Measurement
+     - Metadata
+
+   * - Sales Data
+     - Revenue
+     - Company
+
+   * - Infection Rates
+     - Amount of People Infected
+     - Location
+
+Create a Time Series Collection
+-------------------------------
+
+To create a time series collection, pass the following parameters to the
+`CreateCollection() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.IMongoDatabase.CreateCollection.html>`__
+method:
+
+- The name of the new collection to create
+- The `TimeSeriesOptions <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.TimeSeriesOptions.html>`__  for creating the collection in a `CreateCollectionOptions <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.CreateCollectionOptions.html>`__ object
+
+.. literalinclude:: /includes/fundamentals/code-examples/TimeSeries.cs
+   :start-after: begin-time-series
+   :end-before: end-time-series
+   :language: csharp
+   :dedent:
+
+.. important::
```

Comment:
S: I think it makes sense to move this admonition to the top of this section (line 45 ish), just so the reader knows the server requirement from the beginning